spec/support/face_book_omniauth.rb

```diff
@@ -0,0 +1,14 @@
+module FaceBookOmniauth
+  def self.omniauth_info
+    omni_config = OmniAuth.config
+    omni_config.test_mode = true
```

Comment:
so this is a fixture helper or config?
separate those

Reply:
I guess this is a whole mock for FB